Cargo.toml

```diff
@@ -8,14 +8,14 @@ edition = "2021"
 resolver = "2"
 
 members = [
+    "alloy-compat",
+    "alloy-compat"
+,
```

Comment:
```suggestion
    "alloy-compat",
```

Reply:
done in [b8b6804](https://github.com/EspressoSystems/espresso-network/pull/2848/commits/b8b6804089b90200a932e0012c0390c4b5f3ed50)